server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/deltas.ts

```diff
@@ -92,8 +93,8 @@ export function create(
 	router.get(
 		"/raw/:tenantId/:id",
 		validateRequestParams("tenantId", "id"),
+		throttle(perTenantThrottler, winston, tenantThrottleOptions),
```

Comment:
Can we not call tenantThrottle() middleware here?

Reply:
Changed to perTenantGeneralThrottler, and this throttler is not enabled. :)